systemd/systemd.go

```diff
@@ -210,8 +210,10 @@ func New(rootDir string, mode InstanceMode, rep reporter) Systemd {
 // NewEmulationMode returns a Systemd that runs in emulation mode where
 // systemd is not really called, but instead its functions are emulated
 // by other means.
-func NewEmulationMode() Systemd {
-	return &emulation{}
+func NewEmulationMode(rootDir string) Systemd {
```

Comment:
this probably should have the same code we have in other place that if  rootdir == "" then use dirs.GlobalRootDir

Reply:
Done.